lib/core/src/lib/viewer/components/viewer-render/viewer-render.component.ts

```diff
@@ -141,8 +141,8 @@ export class ViewerRenderComponent implements OnChanges, OnInit {
     extensionsSupportedByTemplates: string[] = [];
     extension: string;
     internalFileName: string;
-    viewerType: string = 'unknown';
```

Comment:
changing type here makes no sense, as in all other places you are operating with a string and typecasting it